### PR TITLE
project proposal & dev update

### PR DESCRIPTION
**Pull request by Tomi-3-0**

### CODE REVIEW

File: projects/eip-7732-implementation-in-nimbus [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Project Template
+
+`eip-7732: enshrined Proposer Builder Separation` implementation in the Nimbus Consensus Client
+
+## Motivation
+
+Implementing proposer-builder separation (PBS) directly into the Nimbus consensus client helps prevent transaction censorship at the protocol level. This enshrines PBS on the protocol, which currently occurs as a sidecar in the protocol, creating a trustless system between proposers and builders. Additionally, it supports scaling Ethereum by enabling future upgrades like Danksharding and Inclusion lists. Working on this project will further solidify my understanding of the core Ethereum protocol, advancing my knowledge from intermediate to advanced.
```
Comment (taxmeifyoucan) [inline projects/eip-7732-implementation-in-nimbus:7]: ePBS is not a dependency of danksharding. You can find some motivation to highlight in epf wiki https://epf.wiki/#/wiki/research/PBS/ePBS 
```suggestion
Implementing proposer-builder separation (PBS) directly into the Nimbus consensus client helps prevent transaction censorship at the protocol level. This enshrines PBS on the protocol, which currently occurs as a sidecar in the protocol, creating a trustless system between proposers and builders.  Working on this project will further solidify my understanding of the core Ethereum protocol, advancing my knowledge from intermediate to advanced.
```

**Review by taxmeifyoucan — COMMENTED** [review]
Thanks for the project proposal!

**Comment by taxmeifyoucan** [comment]
Is it ready to merged? If so, please change it from draft to open

**Comment by Tomi-3-0** [comment]
Hello, @taxmeifyoucan. Comments have been addressed. Thank you

**Comment by Tomi-3-0** [comment]
> Is it ready to merged? If so, please change it from draft to open

It's ready now

File: projects/epbs-implementation-in-nimbus.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+All major changes will be happening in the Consesnus client as specified in this [eip](https://eips.ethereum.org/EIPS/eip-7732). 
+
+## Roadmap
+This below schedule might not hold strictly and the proposed two-weeks-per-component schedule might prove difficult, and I suspect it might prove necessary to interleave things more and difficult to try to plan too tightly. However, the main challenges are listed in this roadmap:
+
+- _Week 5-6_: Understand the current Nimbus-eth2 architecture with the [`storeBlock`](https://github.com/status-im/nimbus-eth2/blob/unstable/beacon_chain/gossip_processing/block_processor.nim `storeBlock`) procedure as a good entry point while getting familiar with the Nim language<br>
+- _Week 7-8_: Implement beacon change specs and write tests <br>
+- _Week 9-10_: Implement the fork-choice specs and write tests <br>
+- _Week 11-12_: Implement the p2p specification changes for ePBS and write tests <br>
+- _Week 13-14_: Implement the honest validator and builder specs with accompanying tests <br>
```
Comment (shyam-patel-kira) [inline projects/epbs-implementation-in-nimbus.md:25]: Hey @Tomi-3-0, The spec until here is already written, just wanted to highlight it so you can focus more on implementing PoC and not re-write specs

Comment (Tomi-3-0) [reply projects/epbs-implementation-in-nimbus.md:25]: Do you mean already implemented in Prysm?

Comment (shyam-patel-kira) [reply projects/epbs-implementation-in-nimbus.md:25]: Hey @Tomi-3-0, No the spec here consensus spec is same for all the client implementation, unless there is any change you want to implement in the spec mentioned in EIP-7732, it is safe to follow the spec

Comment (Tomi-3-0) [reply projects/epbs-implementation-in-nimbus.md:25]: > Hey @Tomi-3-0, No the spec (here consensus spec) is same for all the client implementation, unless there is any change you want to implement in the spec mentioned in EIP-7732, it is safe to follow the spec

So, basically this is just a roadmap based on the outlined spec in the eip.
More like a guide for me on what I should be focusing on at a particular week. I'm not reinventing anything

Comment (shyam-patel-kira) [reply projects/epbs-implementation-in-nimbus.md:25]: okay makes sense, just wanted to point out to see if it is intentional or not, All the best!

**Pull request merged**